cloud_governance/main/environment_variables.py

```diff
@@ -126,6 +126,9 @@ def __init__(self):
                                                                                                 '{}')
         self._environment_variables_dict['DAYS_TO_DELETE_RESOURCE'] = int(
             EnvironmentVariables.get_env('DAYS_TO_DELETE_RESOURCE', '7'))
+        # Comma Separated list. Ex: kubernetes.io/cluster,sigs.k8s.io/cluster-api-provider-aws
+        self._environment_variables_dict['CLUSTER_PREFIX'] = EnvironmentVariables.get_env('CLUSTER_PREFIX',
```

Comment:
I think it should be a list:
[ 'kubernetes.io/cluster', 'sigs.k8s.io/cluster-api-provider-aws']

Reply:
Yes, I thought string list would be better.

Reply:
How did you cast str to list ? (ast.literal_eval)

Reply:
`prefixes = cluster_prefix.strip().split(",")`
Using the strip and split

Reply:
Less recommend, can you try to change it ?